client/app/bundles/KudosApp/stores/KudosStore.js

```diff
@@ -0,0 +1,122 @@
+import { findIndex } from 'lodash'
+import { action, computed, observable } from 'mobx'
+import { getKudos, postKudo, patchKudo, postLike, postUnlike } from '../services/KudosService'
+import ColorGenerator from '../utils/colorGenerator'
+
+const DEFAULT_KUDOS_LIST = { kudos: [], total: 0, page: 0 }
+const NOT_FOUND = -1
+
+class KudosStore {
+  @observable currentTab = 'Recent'
+  @observable isFetchingKudos = false
+  @observable kudos = []
+  @observable total = 0
+  @observable currentPage = 0
```

Comment:
@trisarahsorus commenting here as well:

With cursor-based pagination in my PR #34 we'll want to keep track of the cursor (last fetched kudo's created_at and id) rather than `currentPage`